src/styles/Theme.ts

```diff
@@ -90,5 +90,8 @@ export const Theme = createMuiTheme({
     MuiTab: {
       wrapper: { textTransform: 'none' },
     },
+    MuiDialogContent: {
+      root: { padding: '16px 24px' },
```

Comment:
Please make sure this is what we want because it will affect all our dialogs.

Reply:
Good catch, I forgot to remove this when I refactored to the other confirmation modal component. Thanks!